### ISSUE TITLE
Sulfate_mod, Chem_H2O2 convert unit mistake

### ISSUE THREAD

**Issue by CraigYEUNG** [geoschem/geos-chem#2302]
While using GC14.3.0 Aerosol-only simulation, sulfate_mod.F90 use Chem_H2O2 by:
```F90
       DH2O2 = ( PH2O2m(I,J,L) / TS_EMIS * XNUMOL_H2O2 / CM3PERM3 ) &
               * DT / ( ALPHA * M )
       H2O2  = ( H2O20 / ALPHA + DH2O2 )
```
this is to convert unit of kg/m3 to molec/cm3/s

But I found that in dryrun downloaded Prodloss file, Prod_H2O2 unit is already molec/cm3/s. Thus, if still use `XNUMOL_H2O2 / CM3PERM3`, H2O2 concentration will be much higher.

**Comment by yantosca** [comment]
Thanks for writing @CraigYEUNG.  This issue may have crept in during implementation of the KPP chemistry mechanism in the full-chemistry simulation (in v11-01).  All of the prod/loss outputs are in molec/cm3/s as they are generated by KPP.

We can implement a fix for this in a future bugfix version.  For now, you can remove the `XNUMOL_H2O2 / CMPERM3' in your code.  Thanks for alerting us to this!

We only benchmark the full-chemistry simulation, so we wouldn't have caught this.  We look to users like you to provide us with bug reports for aerosol-only and other specialty simulations.

**Comment by yantosca** [comment]
We can now close this issue as PR #2413 is now merged into 14.4.3.
